radio/src/gui/480x272/model_select.cpp

```diff
@@ -239,9 +239,30 @@ void onDeleteModelConfirm(const char * result)
   }
 }
 
+bool confirmModelChange() {
```

Comment:
Is it possible that this function is shared for all LCD dimensions?

Reply:
Sure. Updated